docs/modules/ROOT/pages/index.adoc

```diff
@@ -116,6 +117,15 @@ If you click on it, you will see all of the photos and videos which have been au
 
 image:ios-instant-upload-file-list.png[Photos and videos which were automatically uploaded to ownCloud by Instant Upload.]
 
+== Clear the Cache
+
+image:ios-clear-cache.png[Clear the ownCloud account cache on iOS.]
+
+If you need to delete all the local files within the current account, or a specific account if you have more than one enabled, then click the "_More Options icon_"^1^ next to the relevant account.
+Next, click the second menu item, btn:[Clear cache] ^2^, and the local files within that account will be cleared.
+
+IMPORTANT: Favorite files *won't* be deleted.
```

Comment:
In mobile apps, we use `Available offline` instead of `Favorites`. `Favorites` is a different concept, so that can lead to confusion.

Reply:
Thanks, @jesmrec.

Reply:
So does "_Offline files *won't* be deleted._" fit better?

Reply:
_"Available Offline files won't be deleted"_ will fit the correct app wording

Reply:
Done.